include/container_copy.hpp

```diff
@@ -0,0 +1,52 @@
+class allocator_with_new {
```

Comment:
the whole file does not seem necessary, as it is already generalized in container.hpp